src/styles/app.scss

```diff
@@ -52,7 +70,7 @@ $max-width: 1044px;
 }
 
 *:focus {
-  outline-color: $secondary-colour;
+  outline-color: $lime-green;
```

Comment:
Do you want this to be the `$focus-colour` you defined?

Reply:
yeah, I couldn't get it to work for some reason